core/src/main/java/org/apache/carbondata/core/carbon/querystatistics/QueryStatistic.java

```diff
@@ -45,8 +47,16 @@
    */
   private long startTime;
 
-  public QueryStatistic() {
+  /**
+   * number of count
+   */
+  private long count;
+
+  private String queryIdWthTask;
+
+  public QueryStatistic(String queryId) {
```

Comment:
Why queryId is required in QueryStatistic as queryId will be common for all the query statistics object, previously it was in query statistics recorder class as recorder was common, can we move it to recorder??